src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+/*------------General-----------------------*/
+
+.search {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+}
```

Comment:
why you put this styles on label instead of input?

Reply:
> why you put this styles on label instead of input?

I needed a clickable container around the input.  `display:flex `  because I needed to position the picture and the input. I tried using other display properties but it didn't work right.